extensions/git/package.nls.json

```
@@ -63,6 +63,7 @@
 	"command.showOutput": "Show Git Output",
 	"command.ignore": "Add to .gitignore",
 	"command.revealInExplorer": "Reveal in Side Bar",
+	"command.rebaseAbort": "Rebase Abort",
```

Maybe let's change the order, so it's more Englishy

```suggestion
	"command.rebaseAbort": "Abort Rebase",
```

---

Switched the order.